package.json

```diff
@@ -12,13 +12,17 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "~8.2.13",
+    "@angular/cdk": "^8.2.3",
```

Comment:
**FIX:** Please remove the caret and use specific version. This will make sure that you won't have unintended changes from the dependencies. Read more in:
- https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/
- https://medium.com/the-guild/how-should-you-pin-your-npm-dependencies-and-why-2b8d545c7312

Reply:
All of the things have got fixed as suggested:
- html table got rewritten into the flexbox
- Prettier and Lint config got added